src/api/tests/feature_tests/environment.py

```diff
@@ -114,6 +118,9 @@ def after_scenario(context: Context, scenario: Scenario):
 def after_feature(context: Context, feature: Feature):
     if context.postman_feature:
         context.postman_collection.item.append(context.postman_feature)
+    if context.clear_screen:
+        click.clear()
+        time.sleep(0.5)
```

Comment:
Why do you need a sleep? It's too quick? 💨

Reply:
yeah, it's just to allow the refresh to happen before the next feature kicks off. It's 500ms so barely noticeable